examples/HierarchicalOptimization_examples/BachmannJakStat/loadBachmannParameters.m

```diff
@@ -0,0 +1,187 @@
+            'init_SHP1'
+            'init_STAT5'
+            'offset_CIS_actd'
+            'offset_CIS_cisoe'
+            'offset_CIS_long'
+            'offset_CIS_shp1oe'
+            'offset_CIS_socs3oe'
+            'offset_SOCS3_cisoe'
+            'offset_SOCS3_long'
+            'offset_SOCS3_socs3oe'
+            'offset_pEpoR_actd'
+            'offset_pEpoR_cisoe'
+            'offset_pEpoR_cisoe_pepor'
+            'offset_pEpoR_dr30'
+            'offset_pEpoR_dr7'
+            'offset_pEpoR_fine'
+            'offset_pEpoR_long'
+            'offset_pEpoR_shp1oe'
+            'offset_pEpoR_socs3oe'
+            'offset_pJAK2_actd'
+            'offset_pJAK2_cisoe'
+            'offset_pJAK2_dr30'
+            'offset_pJAK2_dr7'
+            'offset_pJAK2_fine'
+            'offset_pJAK2_long'
+            'offset_pJAK2_shp1oe'
+            'offset_pJAK2_socs3oe'
+            'offset_pSTAT5_actd'
+            'offset_pSTAT5_cisoe'
+            'offset_pSTAT5_conc'
+            'offset_pSTAT5_long'
+            'offset_pSTAT5_shp1oe'
+            'offset_pSTAT5_socs3oe'%;
+            %'sd_pSTAT5_socs3oe'
+            };
+        
+    case 'standard'
+        
+        parameters.name = {...
+            'CISEqc'
+            'CISEqcOE'
+            'CISInh'
+            'CISRNADelay'
+            'CISRNATurn'
+            'CISTurn'
+            'EpoRActJAK2'
+            'EpoRCISInh'
+            'EpoRCISRemove'
+            'JAK2ActEpo'
+            'JAK2EpoRDeaSHP1'
+            'SHP1ActEpoR'
+            'SHP1Dea'
+            'SHP1ProOE'
+            'SOCS3Eqc'
+            'SOCS3EqcOE'
+            'SOCS3Inh'
+            'SOCS3RNADelay'
+            'SOCS3RNATurn'
+            'SOCS3Turn'
+            'STAT5ActEpoR'
+            'STAT5ActJAK2'
+            'STAT5Exp'
+            'STAT5Imp'
+            'init_EpoRJAK2'
+            'init_SHP1'
+            'init_STAT5'
+            'offset_CIS_actd'
+            'offset_CIS_cisoe'
+            'offset_CIS_long'
+            'offset_CIS_shp1oe'
+            'offset_CIS_socs3oe'
+            'offset_SOCS3_cisoe'
+            'offset_SOCS3_long'
+            'offset_SOCS3_socs3oe'
+            'offset_pEpoR_actd'
+            'offset_pEpoR_cisoe'
+            'offset_pEpoR_cisoe_pepor'
+            'offset_pEpoR_dr30'
+            'offset_pEpoR_dr7'
+            'offset_pEpoR_fine'
+            'offset_pEpoR_long'
+            'offset_pEpoR_shp1oe'
+            'offset_pEpoR_socs3oe'
+            'offset_pJAK2_actd'
+            'offset_pJAK2_cisoe'
+            'offset_pJAK2_dr30'
+            'offset_pJAK2_dr7'
+            'offset_pJAK2_fine'
+            'offset_pJAK2_long'
+            'offset_pJAK2_shp1oe'
+            'offset_pJAK2_socs3oe'
+            'offset_pSTAT5_actd'
+            'offset_pSTAT5_cisoe'
+            'offset_pSTAT5_conc'
+            'offset_pSTAT5_long'
+            'offset_pSTAT5_shp1oe'
+            'offset_pSTAT5_socs3oe'
```

Comment:
until here, all names match. probably better readable (and less error prone) if you always set these values and append the scaling and sd parameters and if the hierarchical approach is chosen